app/store/ephemeral_store.js

```diff
@@ -10,8 +12,8 @@ class EphemeralStore {
         this.allNavigationComponentIds = [];
         this.currentServerUrl = null;
         this.safeAreaInsets = {
-            portrait: null,
-            landscape: null,
+            [ViewTypes.PORTRAIT]: null,
```

Comment:
aren't you setting it here as `null`? I think that either ViewTypes.PORTRAIT needs to be defined as `'portrait'` or you revert to your original definition.

also, notice that tests aren't catching this problem because of the `beforeEach` statement